packages/bits/src/lib/convenience/time-frame-bar/time-frame-bar.component.ts

```diff
@@ -41,6 +41,7 @@ import { TimeFramePickerComponent } from "../../time-frame-picker/time-frame-pic
     templateUrl: "./time-frame-bar.component.html",
     styleUrls: ["./time-frame-bar.component.less"],
     changeDetection: ChangeDetectionStrategy.OnPush,
+    host: { "role": "navigation" },
```

Comment:
This is another role that doesn't quite fit the use case here. I'm not sure what it should be, but `navigation` doesn't seem right. `navigation` is supposed to be a group of the main links for a website.
![image](https://user-images.githubusercontent.com/10159693/113601054-79a8ae00-9606-11eb-9dc7-1024dfeac7d3.png)
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Navigation_Role

Reply:
Maybe, spinbutton role be more applicable here